k8s/configmap.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: v1
```

Comment:
This is super cool. However, the main reason I moved it out into its own file is so that it could be easier to manage (bash linting/syntax checking). Is there any way to set the configmap to load a specific file from disk?

Reply:
Yes, once its in a helm chart we will be able to just inject the script from another file and manage it in its own file, but can't do that until we got the helm chart setup. For now this is prep work to allow us to move into a helm chart

Reply:
fair enough. you should probably just delete the other bash script so there isnt duplicate code